test/rubygems/test_gem_version.rb

```diff
@@ -41,6 +41,11 @@ def test_class_create
     assert_equal v('1.1'), Gem::Version.create(ver)
   end
 
+  def test_class_correct
```

Comment:
Usually I prefer `assert` to `assert_equal(true, actual)`, because it seems to me that `assert_equal(true, actual)` is too strict and fragile. I added this test case not because to assure the correctness of my patch but because there was no test case for `Gem::Version.correct?`. But in this case, the point of issue is the return value, so as you said, `assert_equal(true, actual)` might be better for this method.